src/components/GreenButton.js

```diff
@@ -1,6 +1,6 @@
 function GreenButton({message}){
```

Comment:
Since this is of submit type. Might be good to have that be reflective in the name. Perhaps SubmitButton?

Reply:
I renamed the component into SubmitButton.